packages/java/hilla-react/src/main/resources/com/vaadin/flow/server/frontend/dependencies/hilla/components/lit/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hilla-components-dependencies",
```

Comment:
I feel the lit package should not be available in the react module.

Reply:
My idea was to have only one module `hilla` where you have both packages and this modules isn't bound to lit or react, it just have Hilla components. 

Currently, both `hilla` and `hilla-react` actually have the same content, because init-app logic depends on what artifact is used. 

But init-app will be moved from the Hilla plugin, so I expect `hilla-react` module to be completely removed soon (in a separate PR). 

So finally we'll have one single module `hilla` with both packages and Flow decides which package to use based on auto-detection.